crates/swc/tests/tsc-references/logicalNotOperatorWithStringType_es2015.2.minified.js

```diff
@@ -1,3 +1,6 @@
+function foo() {
```

Comment:
Shouldn't we add the cost for `function` + `()` + `{}`?

Reply:
If we decide to do so, we should also check the number of function calls (by multiplying them)

Reply:
Sure! I've already written a separate PR for this optimization, just some details needs to be polished.